101-webapp-including-insights/azuredeploy.json

```diff
@@ -0,0 +1,142 @@
+{
+  "$schema": "https://schema.management.azure.com/schemas/2015-01-01/deploymentTemplate.json#",
+  "contentVersion": "1.0.0.0",
+  "parameters": {
+    "webappName": {
+      "type": "string",
+      "defaultValue": "uniqueString()",
```

Comment:
uniqueString needs to be seeded with something

"defaultValue": "[concat('web-', uniqueString(resourceGroup().id))]"